### DISCUSSION TITLE
Create a baserow and an n8n intégration

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#6994] [Ideas]
Hello,

As you probably know you are among the top nocode open source solution.

It would be really nice if we can have easy integration with baserow and n8n (other super cool nocode solutions)

Thanks a lot

**Comment by melohagan** [comment]
Hey @oswaldB 

We're looking at ways to encourage and reward community for building integrations. 

There is a solid docs page here: https://docs.budibase.com/docs/writing-your-own-external-data-connector

**Reply by Plaidmustache** [reply]
Did anyone ever do this?

**Reply by sdsys-ch** [reply]
Cf. [here](https://docs.budibase.com/docs/n8n)
